settings.txt

```diff
@@ -1,5 +1,5 @@
-H�he: 16
-Breite: 16
-Spieler: 4
+Height: 16
+Width: 16
+Player: 4
```

Comment:
One final minor nit: `Player` -> `Players`.
Sorry for not picking this up before.